### PR TITLE
Added all course logs to student page and fixed calculations

### PR DESCRIPTION
**Pull request by reish981**

### CODE REVIEW

File: src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java [hunk 1]
```diff
@@ -100,42 +105,50 @@ private ModelAndView studentSummary(String sectionId, String studentId, boolean
...
-        summaryForSections.stream()
-                .flatMap(summary -> summary.getEntries().stream())
-                .filter(entry -> entry.getStudentId() == validatedStudentId)
-                .findFirst()
-                .ifPresent(entry ->  addAssignmentSummaryToPage(page, entry, isSimpleAttendance, student, assignment));
+        List<AttendanceSummaryModel.Entry> entries = new ArrayList<>();
+        if (student.getSisUserId() != null) {
+            for (AttendanceSummaryModel model : summaryForSections) {
+                for (AttendanceSummaryModel.Entry entry : model.getEntries()) {
+                    if (student.getSisUserId().equals(entry.getSisUserId())) {
```
Comment (KSU-Github) [inline src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java]: MAJOR Refactor this code to not nest more than 3 if/for/while/switch/try statements. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS134)

**Comment by KSU-Github** [comment]
SonarQube analysis reported 9 issues
* ![CRITICAL][CRITICAL] 1 critical
* ![MAJOR][MAJOR] 2 major
* ![MINOR][MINOR] 6 minor

Watch the comments in this conversation to review them.

#### 8 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [SummaryControllerITest.java#L109](https://github.com/kstateome/lti-attendance/blob/414095ac2320d6444528801fa4671c21123d4334/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L109): Either log or rethrow this exception. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [simpleStudentSummary.jsp#L8](https://github.com/kstateome/lti-attendance/blob/414095ac2320d6444528801fa4671c21123d4334/src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L108](https://github.com/kstateome/lti-attendance/blob/414095ac2320d6444528801fa4671c21123d4334/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L108): Define a constant instead of duplicating this literal "/studentSummary/" 5 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L148](https://github.com/kstateome/lti-attendance/blob/414095ac2320d6444528801fa4671c21123d4334/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L148): Define a constant instead of duplicating this literal "sectionId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L151](https://github.com/kstateome/lti-attendance/blob/414095ac2320d6444528801fa4671c21123d4334/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L151): Define a constant instead of duplicating this literal "studentId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L152](https://github.com/kstateome/lti-attendance/blob/414095ac2320d6444528801fa4671c21123d4334/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L152): Define a constant instead of duplicating this literal "attendances" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L158](https://github.com/kstateome/lti-attendance/blob/414095ac2320d6444528801fa4671c21123d4334/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L158): Define a constant instead of duplicating this literal "dateOfClass" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L166](https://github.com/kstateome/lti-attendance/blob/414095ac2320d6444528801fa4671c21123d4334/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L166): Define a constant instead of duplicating this literal "entries" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by kawesome** [comment]
Can one of the admins verify this patch?

**Comment by KSU-Github** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [SummaryControllerITest.java#L109](https://github.com/kstateome/lti-attendance/blob/730c2052bd002a7adb625498976dfb7c79103a9f/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L109): Either log or rethrow this exception. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [simpleStudentSummary.jsp#L8](https://github.com/kstateome/lti-attendance/blob/730c2052bd002a7adb625498976dfb7c79103a9f/src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L108](https://github.com/kstateome/lti-attendance/blob/730c2052bd002a7adb625498976dfb7c79103a9f/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L108): Define a constant instead of duplicating this literal "/studentSummary/" 5 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L148](https://github.com/kstateome/lti-attendance/blob/730c2052bd002a7adb625498976dfb7c79103a9f/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L148): Define a constant instead of duplicating this literal "sectionId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L151](https://github.com/kstateome/lti-attendance/blob/730c2052bd002a7adb625498976dfb7c79103a9f/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L151): Define a constant instead of duplicating this literal "studentId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L152](https://github.com/kstateome/lti-attendance/blob/730c2052bd002a7adb625498976dfb7c79103a9f/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L152): Define a constant instead of duplicating this literal "attendances" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L158](https://github.com/kstateome/lti-attendance/blob/730c2052bd002a7adb625498976dfb7c79103a9f/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L158): Define a constant instead of duplicating this literal "dateOfClass" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L166](https://github.com/kstateome/lti-attendance/blob/730c2052bd002a7adb625498976dfb7c79103a9f/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L166): Define a constant instead of duplicating this literal "entries" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by killsto** [comment]
Looks like there was data added and removed to the page. The test was halfway changed to reflect that by removing the outdated expectations, but there are not new expectations added for the new data.

The test should probably be kept up to date if it still has value.

**Comment by KSU-Github** [comment]
SonarQube analysis reported 8 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [SummaryControllerITest.java#L109](https://github.com/kstateome/lti-attendance/blob/2595dc9458f77b2df36fe15fff3818f44d100ca2/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L109): Either log or rethrow this exception. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [simpleStudentSummary.jsp#L8](https://github.com/kstateome/lti-attendance/blob/2595dc9458f77b2df36fe15fff3818f44d100ca2/src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L108](https://github.com/kstateome/lti-attendance/blob/2595dc9458f77b2df36fe15fff3818f44d100ca2/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L108): Define a constant instead of duplicating this literal "/studentSummary/" 5 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L148](https://github.com/kstateome/lti-attendance/blob/2595dc9458f77b2df36fe15fff3818f44d100ca2/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L148): Define a constant instead of duplicating this literal "sectionId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L151](https://github.com/kstateome/lti-attendance/blob/2595dc9458f77b2df36fe15fff3818f44d100ca2/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L151): Define a constant instead of duplicating this literal "studentId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L152](https://github.com/kstateome/lti-attendance/blob/2595dc9458f77b2df36fe15fff3818f44d100ca2/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L152): Define a constant instead of duplicating this literal "attendances" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L158](https://github.com/kstateome/lti-attendance/blob/2595dc9458f77b2df36fe15fff3818f44d100ca2/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L158): Define a constant instead of duplicating this literal "dateOfClass" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L166](https://github.com/kstateome/lti-attendance/blob/2595dc9458f77b2df36fe15fff3818f44d100ca2/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L166): Define a constant instead of duplicating this literal "entries" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java [hunk 2]
```diff
@@ -100,42 +109,49 @@ private ModelAndView studentSummary(String sectionId, String studentId, boolean
...
+        }
 
-    private void addAssignmentSummaryToPage(ModelAndView page, AttendanceSummaryModel.Entry entry, boolean isSimpleAttendance, AttendanceStudent student, AttendanceAssignment assignment) {
-        page.addObject("attendanceSummaryEntry", isSimpleAttendance ?
-            new AttendanceSummaryModel.Entry(entry.getCourseId(), entry.getSectionId(), entry.getStudentId(), entry.getSisUserId(), entry.getStudentName(), student.getDeleted(), entry.getTotalClassesTardy(), entry.getTotalClassesMissed(), entry.getTotalClassesExcused(), entry.getTotalClassesPresent())
-            : new AttendanceSummaryModel.Entry(entry.getCourseId(), entry.getSectionId(), entry.getStudentId(), entry.getSisUserId(), entry.getStudentName(), student.getDeleted(), entry.getSumMinutesMadeup(), entry.getRemainingMinutesMadeup(), entry.getSumMinutesMissed(), entry.getPercentCourseMissed()));
+        int totalPresentDays = 0, totalTardyDays  = 0, totalAbsentDays = 0, totalExcusedDays = 0;
+
+        for (AttendanceSummaryModel.Entry entry : entries) {
```
Comment (killsto) [inline src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java]: The total presentDays calculation should probably be moved out of the controller and be unit tested.

**Comment by KSU-Github** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [SummaryControllerITest.java#L109](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L109): Either log or rethrow this exception. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [simpleStudentSummary.jsp#L8](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L108](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L108): Define a constant instead of duplicating this literal "/studentSummary/" 5 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L148](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L148): Define a constant instead of duplicating this literal "sectionId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L151](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L151): Define a constant instead of duplicating this literal "studentId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L152](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L152): Define a constant instead of duplicating this literal "attendances" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L158](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L158): Define a constant instead of duplicating this literal "dateOfClass" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L166](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L166): Define a constant instead of duplicating this literal "entries" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [RosterControllerArquillianTest.java#L22](https://github.com/kstateome/lti-attendance/blob/10ea510fd1dd9481274934c81860a3db4738d49d/src/test/java/edu/ksu/canvas/attendance/controller/arquillian/RosterControllerArquillianTest.java#L22): Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00115)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java [hunk 3]
```diff
@@ -100,42 +109,49 @@ private ModelAndView studentSummary(String sectionId, String studentId, boolean
...
-                .ifPresent(role -> page.addObject("isStudent", true));
+        List<AttendanceSummaryModel.Entry> entries = new ArrayList<>();
 
-        page.addObject("sectionId", sectionId);
-        page.addObject("student", student);
-        page.addObject("summaryForm", makeupForm);
-        return page;
-    }
+        if (student.getSisUserId() != null) {
+            entries = summaryForSections.stream().flatMap(model -> model.getEntries().stream()).filter(entry -> student.getSisUserId().equals(entry.getSisUserId())).collect(Collectors.toList());
```
Comment (killsto) [inline src/main/java/edu/ksu/canvas/attendance/controller/SummaryController.java]: This line is pretty long. Some versions of intellij will autoformat streams for you. If you are on an old version you can add a new line for every call after stream(). 

Sometimes that is excessive so I usually just use my best judgement to make it readable.

**Comment by KSU-Github** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![CRITICAL][CRITICAL] [SummaryControllerITest.java#L109](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L109): Either log or rethrow this exception. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1166)
1. ![MAJOR][MAJOR] [simpleStudentSummary.jsp#L8](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/main/webapp/WEB-INF/jsp/simpleStudentSummary.jsp#L8): Add a 'favicon' declaration in this 'header' tag. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=Web%3APageWithoutFaviconCheck)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L108](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L108): Define a constant instead of duplicating this literal "/studentSummary/" 5 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L148](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L148): Define a constant instead of duplicating this literal "sectionId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L151](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L151): Define a constant instead of duplicating this literal "studentId" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L152](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L152): Define a constant instead of duplicating this literal "attendances" 4 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L158](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L158): Define a constant instead of duplicating this literal "dateOfClass" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [SummaryControllerITest.java#L166](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/test/java/edu/ksu/canvas/attendance/controller/SummaryControllerITest.java#L166): Define a constant instead of duplicating this literal "entries" 3 times. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS1192)
1. ![MINOR][MINOR] [RosterControllerArquillianTest.java#L22](https://github.com/kstateome/lti-attendance/blob/c5ef9d38131fdb652c99a6dc6b43bfd46c9e9c43/src/test/java/edu/ksu/canvas/attendance/controller/arquillian/RosterControllerArquillianTest.java#L22): Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [rule](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AS00115)

[CRITICAL]: https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL'
[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**